src/components/contact/contact-person.tsx

```diff
@@ -0,0 +1,39 @@
+import { theme } from '@bjerk/brand';
+import { Flex, Text } from 'theme-ui';
+import { NetworksLinks } from '../illustrations/networks-links';
+import { Image } from '../image';
```

Comment:
Why not `next-theme-ui`?

```suggestion
import { Image } from 'next-theme-ui';
```